rules/consistent-function-scoping.js

```diff
@@ -4,12 +4,17 @@ const getDocumentationUrl = require('./utils/get-documentation-url');
 const MESSAGE_ID_ARROW = 'ArrowFunctionExpression';
 const MESSAGE_ID_FUNCTION = 'FunctionDeclaration';
 
+function getReferences(scope) {
+	const {references, childScopes} = scope;
+	return references.concat(...childScopes.map(getReferences));
```

Comment:
Is this code against `no-fn-reference-in-iterator`? Why not caught?

Reply:
Yes, it should have been caught. No idea why it wasn't. Can you open an issue for it?